ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Abstract.hs

```diff
@@ -83,9 +111,10 @@ class ( IsLedger l
   --
   -- This is passed the ledger state ticked to the slot of the given block, so
   -- 'applyChainTickLedgerResult' has already been called.
-  applyBlockLedgerResult ::
+  applyBlockLedgerResultWithOpts ::
```

Comment:
I personally do not see the benefit of renaming `applyBlockLedgerResult` to `applyBlockLedgerResultWithOpts`, as we do not have a variant without options.